src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/components/ManagePersonnelToolBar/index.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { IconButton, AddIcon, EditIcon, DeleteIcon } from '@equinor/fusion-components';
+import * as styles from './styles.less';
+
+export type IconButtonProps = {
+    disabled?: boolean;
+    iconColor?: string;
+    onClick?: (e: React.MouseEvent<HTMLButtonElement>) => void;
+};
+
+export type ToolBarProps = {
+    addButton?: IconButtonProps,
+    deleteButton?: IconButtonProps,
+    editButton?: IconButtonProps,
+};
+
+const ManagePersonnelToolBar: React.FC<ToolBarProps> = ({ addButton, deleteButton, editButton }) => {
+    return <div className={styles.container}>
```

Comment:
Is this formatted?

Reply:
what do you mean ?

Reply:
Formatted with Prettier

Reply:
no, the extension was not installed in the "container". 
Didnt know that was a thing.. 
strange